applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_strategy.h

```diff
@@ -30,38 +30,30 @@
 #include "includes/kratos_flags.h"
 #include "geometries/geometry.h"
 #include "includes/element.h"
-#include "solid_mechanics_application.h"
```

Comment:
What this does different of the Newton Raphson for MPM?

Reply:
The MPM_strategy is to generate new material points and to perform search at every time step where the material points are located within the background grids. The NR is to perform the nonlinear iteration.